languages/en.ini

```diff
@@ -248,6 +250,7 @@ Conference Proceeding = "Conference Proceeding"
 Configuration = "Configuration"
 confirm_delete = "Are you sure you want to delete this?"
 confirm_delete_brief = "Delete Item?"
+confirm_delete_feedback = "Delete Feedback"
```

Comment:
Should this have a question mark on it?

Reply:
On comparing to the existing `confirm_delete_tags_brief ` string, I think the answer is actually no. Sorry about that! :-)